SPECS/xcp-ng-generic-lib.spec

```diff
@@ -0,0 +1,57 @@
+Name:           xcp-ng-generic-lib
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        A library of algorithms, I/O and networking functions...
+License:        GPLv3
+URL:            https://github.com/xcp-ng/xcp-ng-generic-lib
+Source0:        https://github.com/xcp-ng/xcp-ng-generic-lib/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+
+%description
+A library of algorithms, I/O and networking functions... used by XCP-ng tools or daemons.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build && cd build
+%cmake3 .. -DCMAKE_INSTALL_PREFIX=%{buildroot}/usr
```

Comment:
can -DCMAKE_INSTALL_PREFIX=%{buildroot}/usr be replaced by the use of %makeinstall in %install, or does cmake need all the installation paths to be defined here?